src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceFileLogger.cs

```diff
@@ -46,7 +48,7 @@ public LinuxAppServiceFileLogger(string logFileName, string logFileDirectory, IF
         public int MaxFileSizeMb { get; set; } = 10;
 
         // Maximum time between successive flushes (seconds)
-        public int FlushFrequencySeconds { get; set; } = 30;
+        public int MaxFlushFrequencySeconds { get; set; } = 30;
```

Comment:
Does it need to be public?

Reply:
I believe that these vars are public for use in Unit Tests. The MaxFlushFrequencySeconds is not currently used anywhere from what I can see. I will change the access to private on this one